bagger/src/bagger.py

```diff
@@ -147,23 +153,37 @@ def load_xml(path):
     return load_from_string(xml_string)
 
 
-def dispatch_bag(bag_details):
-    # now zip this bag in a way that will be efficient for the archiver
-    logging.debug("creating zip file for " + bag_details["b_number"])
+def pack(bag_details):
+    archive_format = settings.ARCHIVE_FORMAT
+
+    logging.debug(" ".join(["packing", archive_format, bag_details["b_number"]]))
 
-    shutil.make_archive(
-        bag_details["zip_file_path"][0:-4], "zip", bag_details["directory"]
+    base_name = "/".join([settings.WORKING_DIRECTORY, bag_details["b_number"]])
```

Comment:
minor: when you're constructing paths, I think it's neater to use
```suggestion
    base_name = os.path.join(settings.WORKING_DIRECTORY, bag_details["b_number"])
```